include/dra.h

```diff
@@ -123,8 +132,8 @@ typedef struct {
     /* 0x70 */ s32 unk70;
     /* 0x74 */ s32 unk74;
     /* 0x78 */ s32 unk78;
-    /* 0x7C */ u16 unk7C;
-    /* 0x7E */ u16 unk7E;
+    /* 0x7C */ unkUnion3 unk7C;
+    /* 0x7E */ u8 unk7E;
```

Comment:
Is the `u8` intentional? It should be either `u16 unk7E;` or `u8 unk7E; u8 unk7F;`

Reply:
yes, because unkUnion3 has three u8s or one u16, in this last case the offset will keep in the correct position because the compiler will fill the alignment pads automatically

Reply:
with s16 unk7E, the real 0x7E was being moved to 0x7F because the u16 didn't fit

Reply:
![image](https://user-images.githubusercontent.com/96613413/196299395-add7afd8-44b3-423d-a541-742830d6d3c0.png)